### PR TITLE
Compose email

### PR DESCRIPTION
**Pull request by FE-Emy**
Story 4 Task 1,2 and 3 ready for review

### CODE REVIEW

File: src/NewEmail/Form/index.jsx
Comment (ashleycoles) [inline src/NewEmail/Form/index.jsx]: You can add a required attribute to each form input here, that will provide some extra validation and prevent the form from being submitted if any field is empty. Keep the check you have in handleSend as well.

File: src/NewEmail/Form/index.jsx
Comment (ashleycoles) [inline src/NewEmail/Form/index.jsx]: These two inputs are the same, you should make a text input component and re-use it for both of these. You can pass in all the data (placeholder, value, onChange function etc) as props.

That way, if you ever need more inputs elsewhere, you can just re-use the component. Also, if you ever need to change the way they look, you only have to change it in one place.

File: src/NewEmail/Form/index.jsx
Comment (ashleycoles) [inline src/NewEmail/Form/index.jsx]: Looking at the API documentation, this url should actually be https://email-client-api.dev.io-academy.uk/emails

File: src/NewEmail/Form/index.jsx
Comment (ashleycoles) [inline src/NewEmail/Form/index.jsx]: You're missing some data in here as per the API documentation:

```
 {
      "name": "Bob Ross",
      "email": "bob.ross@paintings.com",
      "subject": "Example",
      "body": "Exmaple text",
  }
```

Note: The name can just be set to 'Red Pandas' - it's the name of the person sending the email.

File: src/NewEmail/Form/index.jsx
Comment (ashleycoles) [inline src/NewEmail/Form/index.jsx]: You should display this error message on the page, same way you're displaying the sent/cancelled messages. Please can you word it is 'All fields are required to send an email'

File: src/NewEmail/Form/index.jsx
Comment (ashleycoles) [inline src/NewEmail/Form/index.jsx:4]: Add an empty line above the function

File: src/NewEmail/Form/index.jsx
Comment (ashleycoles) [reply src/NewEmail/Form/index.jsx]: Also - can you make these inputs full width please, I think it would be better if they took up the whole width of the page.

**Review by ashleycoles — CHANGES_REQUESTED** [review]
Nice work! There are some changes here for you to work on.

Note: We'll need to hold off on merging this story into main until story 2 is in, and preferably story 3 as well.

Once those have been merged, you will probably need to make a few updates to this branch.

For now, look at the comments I've made, and then we'll look at merging it once we're ready.

File: src/NewEmail/Form/index.jsx
Comment (ghost) [reply src/NewEmail/Form/index.jsx]: All done.

File: src/NewEmail/Form/index.jsx
Comment (FE-Emy) [reply src/NewEmail/Form/index.jsx]: Done.

File: src/Components/SentEmails/index.jsx
Comment (ashleycoles) [inline src/Components/SentEmails/index.jsx]: This fetch request is still requesting all emails, it should be going to /emails/sent

File: src/NewEmail/Form/index.jsx
Comment (ashleycoles) [inline src/NewEmail/Form/index.jsx]: The date here doesn't do anything, the API just ignores it, so you might as well remove it

**Review by ashleycoles — CHANGES_REQUESTED** [review]

**Review by ashleycoles — APPROVED** [review]

**Pull request merged**
